xarray/core/computation.py

```diff
@@ -549,8 +549,8 @@ def apply_variable_ufunc(func, *args, **kwargs):
                              'or load your data into memory first with '
                              '``.load()`` or ``.compute()``')
         elif dask == 'parallelized':
-            input_dims = [broadcast_dims + input_dims
-                          for input_dims in signature.input_core_dims]
+            input_dims = [broadcast_dims + dims
+                          for dims in signature.input_core_dims]
```

Comment:
This is a good change but just so I'm clear, this was just a style correction and doesn't change the behavior, does it?

Reply:
Yes, this is just a style fix. The inner variable doesn't leak, but it's still weird to reuse it.